build/networking/Dockerfile.ubuntu

```diff
@@ -69,14 +69,18 @@ RUN apt-get install -y apt-utils \
 RUN python3 -m pip install --no-cache-dir --upgrade pip && \
     python3 -m pip install --no-cache-dir grpcio && \
     python3 -m pip install --no-cache-dir ovspy \
-    protobuf==3.20.1 \
     p4runtime \
     pyelftools \
     scapy \
     six \
     cmake>=3.15.0 \
     meson==0.59.4 \
-    ninja>=1.8.2
+    ninja>=1.8.2 && \
+    if [ "$BASE_IMG" = "ubuntu:18.04" ] ; then \
+       python3 -m pip install --no-cache-dir protobuf==3.19.4; \
```

Comment:
Per "https://developers.google.com/protocol-buffers/docs/news/2022-05-06", Python language support in Protocol Buffers was changed starting version 3.20.1. If we use 3.19.4 in Ubuntu 18.04, do we see the issue when using ovs-p4ctl utility?

Reply:
Nope, on 18.04 I have validated with this version of protobuf.
No issue has been seen. Able to run simple_l3 use case without any issue.

PR has been raised only after UT'ing changes for both 18.04 and 20.04.